apps/trading-e2e/src/integration/markets-page.feature

```diff
@@ -15,3 +15,10 @@ Feature: Markets page
     Given I am on the markets page
     When I click on "Suspended" market
     Then trading page for "suspended" market is displayed
+
+  Scenario: Accounts displayed when connected to wallet
+    Given I am on the trading page for an active market
+    And I connect to Vega Wallet
+    When I click on accounts tab
+    Then accounts are displayed
+    And I can see account for tEURO
```

Comment:
I think it should display any asset that has been deposited. I haven't got Vega there because I've had issues trying to get test Vega from the faucet but there's no reason we cant change the test to vega if we deposit some to the test wallet.

Reply:
Na what I mean is that deposited assets go into the general account and are only persisted if replaying the chain. Things from eth for staking go into a different account and are persisted on chain restart, to avoid having to redeposit these assets. Obviously capsule should make this redundant but a potential fix. Not a blocker but if it's an easy win then would be good to get in to prevent ahving to reload test wallet 👍

Reply:
Gotcha, for some reason I can't stake on testnet right now to test this - all the validators are empty as well. Once I can stake and see it showing in the accounts I'll create a follow up ticket to update the test. 👍🏼

Reply:
Dont we want to just mock the api request until we have capsule anyway?

Reply:
I'd like to keep this not-mocked for now since it's not dependant on active markets and simple to set up if testnet gets reset.